pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -12,6 +12,7 @@ const (
 	SingletonClusterName                             = "cluster"
 	InternetReachableFromMaster status.ConditionType = "InternetReachableFromMaster"
 	InternetReachableFromWorker status.ConditionType = "InternetReachableFromWorker"
+	MachineSupported            status.ConditionType = "MachineSupported"
```

Comment:
let's avoid wording Supported/Unsupported for now as this is partially customer facing.  `Compliant`?